helm/charts/determined/values.yaml

```diff
@@ -233,6 +240,10 @@ checkpointStorage:
 # the greatest common divisor of all the sizes (4, in that case).
 maxSlotsPerPod:
 
+
+# TODO( POPULATE THIS COMMENT)
```

Comment:
Again, another remnant from the WIP branch  
```suggestion
# Flag for determining whether or not to create the cluster-scoped objects, when deploying multiple namespaces within the same cluster, should be set to false.
```

Reply:
It looks like this might be redundant https://github.com/determined-ai/determined/blob/0159cedd7fc6c5aa94de5bbe8312edd36afdd09f/helm/charts/determined/values.yaml#L61

Reply:
We use `createNonNamespacedObjects` for the cluster deployment script when deciding whether to create [priority classes](https://github.com/determined-ai/determined/blob/5cb7927389125a1ff28fc4772af01997c6065f47/helm/charts/determined/templates/priority-classes.yaml#L1)

Reply:
Gah, update on this: I didn't realize I redefined that field, we need `createNonNamespacedObjects: true` for the longrunning tests since they spinup a new cluster. Resolved [here](https://github.com/determined-ai/determined/pull/9114)!